cmd/gardenlet/app/app.go

```diff
@@ -73,10 +74,12 @@ import (
 	"github.com/gardener/gardener/pkg/gardenlet/controller"
 	gardenerhealthz "github.com/gardener/gardener/pkg/healthz"
 	"github.com/gardener/gardener/pkg/resourcemanager/controller/garbagecollector/references"
+	"github.com/gardener/gardener/pkg/resourcemanager/predicate"
 	"github.com/gardener/gardener/pkg/utils"
 	"github.com/gardener/gardener/pkg/utils/flow"
 	gardenerutils "github.com/gardener/gardener/pkg/utils/gardener"
 	kubernetesutils "github.com/gardener/gardener/pkg/utils/kubernetes"
+	clientutils "github.com/gardener/gardener/pkg/utils/kubernetes/client"
```

Comment:
Nit: for consistency with the other import aliases for the same import>

```suggestion
	utilclient "github.com/gardener/gardener/pkg/utils/kubernetes/client"
```